dashboard.csss-ci.com/server/api/controllers/rest/v1/Proposals.php

```diff
@@ -274,6 +326,9 @@ public function myProposals_get($id = null) {
         }
     }
 
+    /**
+     * Gets association id from the households model 
```

Comment:
needs more context. 
finds all households related to association => finds all the needs of those households => find the proposals related to the needs